src/components/cc-img/cc-img.stories.js

```diff
@@ -22,24 +22,24 @@ const conf = {
 
 export const defaultStory = makeStory(conf, {
   items: [
-    { text: 'OMG' },
-    { text: 'OMG', skeleton: true },
-    { text: 'OMG', skeleton: true, src: 'http://placekitten.com/200/200' },
+    { accessibleName: 'OMG' },
+    { skeleton: true },
```

Comment:
**suggestion:** I remember we removed `text` / `accessibleName` here to see if the component behaved well when the prop is `undefined` / `null`.

Do we want to create a specific story for that maybe?

A copy of the `defaultStory` with no `accessibleName` in the items?